examples/components/NER/aggregatemodelweights/spec.yaml

```diff
@@ -0,0 +1,39 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
```

Comment:
Is it possible to just reuse weights aggregation script from the MNIST directory?

Reply:
I will fix that part in my [upcoming PR](https://github.com/Azure-Samples/azure-ml-federated-learning/pull/182), after this one has gone in